element_deeplabcut/model.py

```diff
@@ -558,18 +569,18 @@ class PoseEstimationTask(dj.Manual):
     Attributes:
         VideoRecording (foreign key): Video recording key.
         Model (foreign key): Model name.
-        task_mode (load or trigger): Optional. Default load. Or trigger computation.
-        pose_estimation_output_dir ( varchar(255) ): Optional. Output dir relative to
-                                                     get_dlc_root_data_dir.
+        task_mode (load or trigger): One of 'load' (load computed analysis results) or 'trigger' (trigger computation).
+        pose_estimation_output_dir (str): Output directory of pose estimation results relative to
+                                                     the root data directory.
         pose_estimation_params (longblob): Optional. Params for DLC's analyze_videos
                                            params, if not default."""
 
     definition = """
     -> VideoRecording                           # Session -> Recording + File part table
     -> Model                                    # Must specify a DLC project_path
     ---
+    pose_estimation_output_dir: varchar(255)    # Output directory of pose estimation results relative to the root directory
     task_mode='load' : enum('load', 'trigger')  # load results or trigger computation
-    pose_estimation_output_dir='': varchar(255) # output dir relative to the root dir

```

Comment:
Changing the ordering of attributes is technically a change in the table definition. I'd recommend not to make this change unless there's a good reason to.
And, we don't need the default empty string value anymore?